scripts/west_commands/runners/minichlink.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2024 Google LLC.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""CH32V00x specific runner."""
```

Comment:
Is this **CH32V00x specific runner.** true? Maybe it works for all SoC from this manufacturer, yes?

Reply:
https://github.com/cnlohr/ch32v003fun/tree/master/minichlink is the upstream repository.

It works for CH32V003 for sure but it might not support all SoC(s) from WCH it seems.

CC @cnlohr for the definitive answer here.

Reply:
found references of `minichlink` and other parts, I say just rename it to be generic